.dotnet/src/Custom/Chat/ChatCompletionOptions.cs

```diff
@@ -112,6 +112,23 @@ public ChatCompletionOptions()
     [CodeGenMember("ParallelToolCalls")]
     public bool? ParallelToolCallsEnabled { get; set; }
 
+    /// <summary>
+    /// An object specifying the format that the model must output.
+    /// </summary>
+    /// <remarks>
+    /// <p>
+    /// Compatible with GPT-4o, GPT-4o mini, GPT-4 Turbo and all GPT-3.5 Turbo models newer than gpt-3.5-turbo-1106.
+    /// </p>
+    /// <p>
+    /// Learn more in the Structured Outputs guide.
+    /// </p>
+    /// </remarks>
+    [CodeGenMember("ResponseFormat")]
+    public ChatResponseFormat ResponseFormat { get; set; }
```

Comment:
Could you add a CUSTOM note explaining the customization here? Is it just the doc comment?

Reply:
Not just the doc comment; directly generated, there's no separation between response format types used in Assistants and Chat and no discriminator relationship established -- `ChatResponseFormat` in .tsp is contrived (in custom.tsp) and the hierarchy is copied/transferred to preserve separation and produce structure.

With no massaging, this would have just been BinaryData; with an injected type, it would have been a shared "InternalOmniTypedResponseFormat."

Reply:
Actually, I take this back -- it was very much true early on, but I must have "fixed" the spec (customized the ingestion more) along the way, as the model itself now does use the proper, contrived `ChatResponseFormat`. I'll remove the property as suggested.